src/wic.py

```diff
@@ -21,39 +23,102 @@ def is_up_to_date(input_file_path: str, output_file_path: str) -> bool:
 
-    input_file_path = "data_folder/illinois_wic_data_january_2021.pdf"
-    output_file_path = "final_jsons/wic.csv"
 
-    if always_run or not is_up_to_date(input_file_path, output_file_path):
+def read_wic_data(always_run: bool = False) -> WICParticipation:
+
+    input_file_path = "data_folder/illinois_wic_data_january_2021.pdf"
+    women_output_csv_path = "final_jsons/wic_participation_women.csv"
+    infants_output_csv_path = "final_jsons/wic_participation_infants.csv"
+    children_output_csv_path = "final_jsons/wic_participation_children.csv"
+    total_output_csv_path = "final_jsons/wic_participation_total.csv"
+
+    if always_run or \
+       not is_up_to_date(input_file_path, women_output_csv_path) or \
+       not is_up_to_date(input_file_path, infants_output_csv_path) or \
+       not is_up_to_date(input_file_path, children_output_csv_path) or \
+       not is_up_to_date(input_file_path, total_output_csv_path):
         # PDF has 97 pages. Skip page 0 because it shows Statewide totals
         # which we don't need
-        parse_wic_pdf(
+        participation: WICParticipation = parse_wic_pdf(
             input_file_path,
-            output_file_path,
             1,
             96)
 
+        participation.women.to_csv(women_output_csv_path, index=False)
+        participation.children.to_csv(children_output_csv_path, index=False)
+        participation.infants.to_csv(infants_output_csv_path, index=False)
+        participation.total.to_csv(total_output_csv_path, index=False)
+        return participation
+    else:
+        return WICParticipation(
+            women=read_csv(women_output_csv_path),
+            infants=read_csv(infants_output_csv_path),
+            children=read_csv(children_output_csv_path),
+            total=read_csv(total_output_csv_path))
+
+
+def read_csv(path: str) -> pd.DataFrame:
+    return pd.read_csv(path, index_col="fips")
+
+
+def read_json(path: str) -> pd.DataFrame:
+    return pd.read_json(path, orient="index", dtype={"fips": str})
+
+
+def dataframe_from_rows(rows: List[List[str]]) -> pd.DataFrame:
+    columns = ["fips",  # County fips code
+               "NAME",  # County name
+               "race_amer_indian_or_alaskan_native",  # Amer. Indian or Alaskan Native # noqa: E501
+               "race_asian",  # Asian
+               "race_black",  # Black or African American
+               "race_native_hawaii_or_pacific_islander",  # Native Hawaii or Other Pacific Isl. # noqa: E501
+               "race_white",  # White
+               "race_multiracial",  # Multi-Racial
+               "total",  # Total Participants
+               "hispanic_or_latino"]  # Hispanic or Latino
+    df = pd.DataFrame(data=rows, columns=columns)
+    return df.set_index('fips')
+
+
+def extract_columns_from_line(line: str) -> List[int]:
+    # Split out a list like ["Total", "Infants", "1", "2", "3", "4"]
+    str_list = line.split(sep=" ")
+    return [int(s) for s in str_list[2:]]
```

Comment:
I get an error here `ValueError: invalid literal for int() with base 10: '1,044'`

Reply:
Tried to fix it with:
```
def extract_columns_from_line(line: str) -> List[int]:
    # Split out a list like ["Total", "Infants", "1", "2", "3", "4"]
    str_list = line.split(sep=" ")
    non_decimal = re.compile(r'[^[0-9].]+')
    return [int(non_decimal.sub('', s)) for s in str_list[2:]]
```
But I got the same error 🤔

Reply:
I fixed it by doing this:
`return [int(s.replace(",", "")) for s in str_list[2:]]`